conductr_cli/bndl_create.py

```diff
@@ -104,25 +108,27 @@ def bndl_create(args):
                 return 2
 
             process_oci = True
-        elif args.format == 'bundle':
-            peek = buff_in.peek(BNDL_PEEK_SIZE) if args.source is None else None
+        elif args.format == 'bundle' or args.format == 'configuration':
```

Comment:
We distinguish now between `bundle` and `configuration`. For a `configuration` format we do not check if all required properties are part of the `bundle.conf` that we produce or update.

Note that the `configuration` format always needs to be specified explicitly. The format auto-detection will fallback to `bundle` given this is the default use case.